verma2021explaining/openreview/content.tex

```diff
@@ -0,0 +1,293 @@
+$t_{0 \to i} = e^{\gamma_i} \odot x + \delta_i$
+The gammas($\gamma_s$) are truncated just like the deltas and their $L1$ norm is added to the regularization term. Note that these transformations are strictly more expressive. If $\gamma$ is the zero vector, these transformations reduce to regular TGT. 
+\subsubsection{UCI Heart Disease and UCI Boston Housing Dataset}
+In figure \ref{fig:pt_heart_housing} we see the results of our replication on the UCI Boston Housing and UCI Heart Disease dataset. For the UCI Boston Housing data, the TGT method seems to slightly outperform DBM both with and without scaling. This supports claim 1. The deltas($\delta_s$) and gammas($\gamma_s$) show high similarity, supporting claim 2. For the UCI Heart Disease dataset, we do not see a difference in performance without scaling while TGT with scaling performs slightly worse.
+\begin{figure}[H]
+    \includegraphics[width=0.24\textwidth]{images/ptfigures/housing-tradeoff.png}
+    \includegraphics[width=0.244\textwidth]{images/ptfigures/housing-scaling-tradeoff.png}
+    \includegraphics[width=0.24\textwidth]{images/ptfigures/heart-tradeoff.png}
+    \includegraphics[width=0.24\textwidth]{images/ptfigures/heartscaling-tradeoff.png}
+    \caption{Results for the PyTorch replication for UCI Boston Housing and UCI Heart Disease dataset.}
+    \label{fig:pt_heart_housing}
+\end{figure}
+
+\subsubsection{Breast Cancer Wisconsin (Diagnostic) and Pima Indians Diabetes Database}
+In figure \ref{fig:pt_diabetes_breast} we see the results for the Pima Indians Diabetes and Breast Cancer Wisconsin (Diagnostic) dataset. For the diabetes dataset, TGT with and without scaling outperforms DBM when more than one feature is used. This supports claim 1. Since the delta ($\delta$) similarity is close to $1$, claim 2 is also supported. For the Breast Cancer dataset, we see similar performance for DBM and TGT and slightly worse performance with scaling. The deltas ($\delta_s$) still have high similarity, supporting claim 2. 
+
+\begin{figure}[H]
+    \includegraphics[width=0.25\textwidth]{images/ptfigures/diabetes-tradeoff.png}
+    \includegraphics[width=0.24\textwidth]{images/ptfigures/diabetes_scaling-tradeoff.png}=
+    \includegraphics[width=0.24\textwidth]{images/ptfigures/breast-tradeoff.png}
+    \includegraphics[width=0.24\textwidth]{images/ptfigures/breastscaling-tradeoff.png}
```

Comment:
The compilation issues are due to invalid file paths. Since `make` compilation occurs from journal folder, these file paths should reflect that. Specifically, change all paths to be relative to journal, for example: `../openreview/images/ptfigures/diabetes-tradeoff.png`